fh-sync-server-DEVELOPMENT.yaml

```diff
@@ -343,27 +331,16 @@ objects:
       name: fh-sync-server
       weight: 100
     wildcardPolicy: None
-  status:
-    ingress:
-    - conditions:
-      - lastTransitionTime: 2017-07-18T15:32:16Z
-        status: "True"
-        type: Admitted
-      host: fh-sync-server-myproject.127.0.0.1.nip.io
-      routerName: router
-      wildcardPolicy: None
+
 - apiVersion: v1
+  kind: Secret
   data:
     database-admin-password: dUxrSmZQbXNqOFh4dmNpcg==
```

Comment:
probably a follow up issue, but we should generate these values rather than hardcoded